lib/app.jsx

```diff
@@ -99,6 +99,11 @@ const isElectron = ( () => {
 	return () => foundElectron;
 } )();
 
+const isElectronMac = () => {
+	return isElectron() &&
+		get( window, 'process.platform', '' ) === 'darwin';
+}
```

Comment:
> I hope that you tested this since I didn't.

Yes, I did test it. In the web version on OS X the check fails and the styles aren't applied. 👍